tests/toranj/wpan.py

```diff
@@ -216,12 +218,12 @@ class Node(object):
     _SPEED_UP_FACTOR = 1    # defines the default time speed up factor
 
     # path to `wpantund`, `wpanctl`, `ot-ncp-ftd`,`ot-ncp` and `ot-ncp-radio`
-    _WPANTUND = '/usr/local/sbin/wpantund'
```

Comment:
Prefer to keep the path explicit here to be sure about the version being used.
e.g., on my local machine, I have multiple instances of `wpantund` (different plugins) and the one on path is usually not the one we want for the `toranj` test.

Reply:
I think using the one in path is more feasible. Even we have multiple installations, we can override PATH to choose the right one. However, if use fixed path here, everyone must install it under the only path. And the path is different from installation in OTBR /usr. As a result, If we develop OTBR and OpenThread, we must install two wpantund.